resources/assets/js/components/UnclaimedJobCard.tsx

```diff
@@ -1,11 +1,10 @@
 import React from "react";
 import { FormattedMessage, useIntl } from "react-intl";
 import { JobStatus } from "../models/lookupConstants";
-import { Link } from "../models/app";
 import { jobStatus } from "../models/localizedConstants";
 
 export interface UnclaimedJobCardProps {
-  jobLink: Link;
+  jobTitle: string;
   createdAt: string;
   status: JobStatus;
   hiringManagers: string[];
```

Comment:
Its only possible for a job to have a single manager at a time.

Reply:
In the wireframe there were multiple. However, if that's the case then I can remove all of the `hiringManagers.map()` stuff.